howtos/custom_generators.rst

```diff
@@ -3,98 +3,30 @@
-overridden as it doesn't have to define any include paths or library paths.
+    Check the reference of the :ref:`custom_generator` section to know the syntax and attributes available.
 
-If you want to create a generator that creates more than one file, you can leave the ``filename()`` empty, and return a dictionary of
-filenames->contents in the ``content()`` method:
+Creating a Premake generator
+----------------------------
 
-.. code-block:: python
-
-    class MultiGenerator(Generator):
-
-        @property
-        def content(self):
-            return {"filename1.txt": "contents of file1",
-                    "filename2.txt": "contents of file2"}  # any number of files
-
-        @property
-        def filename(self):
-            pass
-
-Once, it is defined in the ``conanfile.py`` you can treat is as a regular package, typically you
-will ``export`` it first to your local cache, test it, and once it is working fine, you would
-``upload`` it to a server.
-
-
-You have access to the ``conanfile`` instance at ``self.conanfile`` and get information from the requirements:
-
-+-----------------------------------------+------------------------------------------------------------------------------------------------+
-| Variable                                | Description                                                                                    |
-+=========================================+================================================================================================+
-| self.conanfile.deps_cpp_info            | :ref:`deps_cpp_info<deps_cpp_info_attributes_reference>`                                       |
-+-----------------------------------------+------------------------------------------------------------------------------------------------+
-| self.conanfile.deps_env_info            | :ref:`deps_env_info<deps_env_info_attributes_reference>`                                       |
-+-----------------------------------------+------------------------------------------------------------------------------------------------+
-| self.conanfile.deps_user_info           | :ref:`deps_user_info<deps_user_info_attributes_reference>`                                     |
-+-----------------------------------------+------------------------------------------------------------------------------------------------+
-| self.conanfile.env                      | dict with the applied env vars declared in the requirements                                    |
-+-----------------------------------------+------------------------------------------------------------------------------------------------+
-
-Premake generator example
--------------------------
-
-Create a project (the best is a git repository):
+Create a folder with a new *conanfile.py* with the following contents::
 
 .. code-block:: bash
 
    $ mkdir conan-premake && cd conan-premake
```

Comment:
This is not rendering ok:

![image](https://user-images.githubusercontent.com/1406456/56948225-c4bf0a00-6b2f-11e9-9896-44493415390d.png)